docs_src/microservices/application/details/EnvironmentVariables.md

```diff
@@ -0,0 +1,19 @@
+---
+title: App Services - Environment Variables
+---
+
+# Application Services - Environment Variables
+
+See the [Common Environment Variables](../../configuration/CommonEnvironmentVariables.md) section for the list of environment variables common to all EdgeX Services. The remaining in this section are specific to Application Services.
+
+### EDGEX_SERVICE_KEY
+
+This environment variable overrides the [`-sk/--serviceKey` command-line option](CommandLine.md#service-key) and the default set by the application service.
```

Comment:
is there a profile command line option? should that be mentioned here too?

Reply:
Yes, but it is part of the common command line options. 
https://docs.edgexfoundry.org/3.1/microservices/configuration/CommonCommandLineOptions/#profile

Reply:
ah ok. so we are just touching on the ones for app services.

Reply:
Yep, there is a reference to the common ones